AMW_angular/io/src/app/shared/date-picker/date-picker.component.html

```diff
@@ -0,0 +1,37 @@
+<div *ngIf="errorMessage" class="alert alert-warning alert-dismissible" role="alert">
```

Comment:
Could this be replaced with the new control flow. There are other `*ngIf`s in this component too!

Reply:
Copy paste classic